### DISCUSSION TITLE
Seeding the production database

### DISCUSSION THREAD

**Discussion by kentcdodds** [epicweb-dev/epic-stack#326] [Ideas]
I need to start thinking about how the Epic Stack will recommend seeding data in production. Anyone have workflows they're happy with? I've got some ideas, but I'd love to hear from others first!

**Comment by stephen776** [comment]
Quick and dirty might be generating an empty Prisma migration and adding sql statements to insert the needed records.

**Comment by sys13** [comment]
Consideration for multi-tenant apps: when a new tenant is added there's some default data that needs to be added, that can later be customized per tenant.

**Comment by kentcdodds** [comment]
I've written the docs: https://github.com/epicweb-dev/epic-stack/blob/main/docs/deployment.md#seeding-production
